Rmd/tgx_ddi.Rmd

```diff
@@ -66,6 +66,7 @@ params:
   alpha: 0.05
   feature_id: null
   biospyder: null
+  prefix: null # Prefix for output files
```

Comment:
How is this accomplished in the other reports again?

Reply:
For all the reports, a prefix gets made with get_prefix() then used to name the report output file within the write_reports.R script. I saved it as a param in this case so the prefix could be used in the Rmarkdown script to name the csv output file I want created.
BUT the simpler way would be to just name the files using reports_filter, aka the facet name, so I'm going to add a commit changing it to do that. Thanks.